recipe/meta.yaml

```diff
@@ -20,27 +20,31 @@ build:
 requirements:
   host:
     - pip
-    - hatchling >=0.25
-    - python >=3.7
+    - hatchling >=1.5
+    - python >=3.8
   run:
     - entrypoints
     - jupyter_core >=4.9.2
     - nest-asyncio >=1.5.4
-    - python >=3.7
+    - python >=3.8
     - python-dateutil >=2.8.2
     - pyzmq >=23.0
     - tornado >=6.2
     - traitlets
+    - importlib_metadata >=4.8.3
+    - jupyter_core >=4.12,!=5.0.*
+    - python-dateutil >=2.8.2
+    - pyzmq >=23.0
+    - tornado >=6.2
+    - traitlets >=5.3
 
 test:
   requires:
-    - async_generator
-    - ipykernel >=6.12
-    - ipython
-    - mock
-    - pip
+    - coverage
+    - ipykernel >=6.14
+    - paramiko; sys_platform == 'win32'
```

Comment:
Would just be `- paramiko  # [win]`. but really, as a test dep, it's fine to just bring it along.